apps/server/src/routes/v1/users.ts

```diff
@@ -363,3 +292,75 @@ router.get(
   authorizationMiddleware,
   getUpcomingUserActivities
 )
+
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     Income:
+ *       type: object
+ *       properties:
+ *         date:
+ *           type: string
+ *           format: date
+ *         amount: string
+ *     Wallets:
+ *       type: object
+ *       properties:
+ *         id:
+ *           type: string
+ *         userId:
+ *           type: string
+ *         totalAmount:
+ *           type: number
+ *         totalIncome:
+ *           type: number
+ *         earnings:
+ *           type: array
+ *           items:
+ *             $ref: '#/components/schemas/Income'
+ *         type:
+ *           type: string
+ *         createdAt:
+ *           type: string
+ *           format: date-time
+ *         updatedAt:
+ *           type: string
+ *           format: date-time
+ * /api/v1/users/personal-wallets:
+ *   get:
+ *     summary: Get user personal svings wallet
+ *     description: Retrieves user's personal svings wallet .
+ *     tags: [Users]
+ *     security:
+ *       - BearerAuth: []
+ *     responses:
+ *       200:
+ *         description: Success. Returns a list of upcoming user activities.
```

Comment:
The description should be related to the fetched wallets, not the user's upcoming activities.